zabbix_auto_config/__init__.py

```diff
@@ -11,11 +11,13 @@
 import multiprocessing_logging
 import tomli
 
-from .__version__ import __version__
 from . import exceptions
 from . import models
 from . import processing
 
+import importlib.metadata
```

Comment:
I guess this import should be placed up with the other stdlib imports?

Reply:
that makes sense; done :+1: